content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -234,6 +234,17 @@ This limitation surfaces in multi-socket, as well as single-socket multi NUMA sy
 and can cause significant performance degradation in latency-critical execution and high-throughput applications if the
 Topology Manager decides to align resources on non-adjacent NUMA nodes.
 
+* `max-allowable-numa-nodes` (beta, visible by default).
+The `max-allowable-numa-nodes` policy option is beta in Kubernetes {{< skew currentVersion >}}.
+
+With the `TopologyManager` enabled, the time to admit a pod is tied to the number of NUMA nodes on the physical machine.
+With the new `max-allowable-numa-nodes` option, nodes with more than 8 NUMA nodes can be alowed to run
+with the `TopologyManager` enabled. However, it is unknown exactly how much this will slow
+down pod admission on any given system. This feature is therefore to be used at-your-own-risk until we have
+a proper solution in place to reduce the state explosion that causes pod admission time to slow down as the
+number of NUMA nodes increases. The value of `max-allowable-numa-nodes` does not (in and of itself) affect the
+latency of pod admission.
```

Comment:
```suggestion
The time to admit a pod is tied to the number of NUMA nodes on the physical machine.
If you select the the `max-allowable-numa-nodes` policy option, nodes with more than 8 NUMA nodes can
be allowed to run with the topology enabled. The Kubernetes project only has limited data of the impact
of using the topology manager on (Kubernetes) nodes with more than 8 NUMA does. Because of that
lack of data, using this policy option is **not** recommended and is at your own risk.

Setting a value of `max-allowable-numa-nodes` does not (in and of itself) affect the
latency of pod admission, but binding a Pod to a (Kubernetes) node with many NUMA does does
have an impact.
Future, potential improvements to Kubernetes may improve Pod admission performance and the high
latency that happens as the number of NUMA nodes increases.
```

We usually [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future); this one is couched so that it aligns with the spirit of that guidance.